iothub/device/src/Pipeline/RetryDelegatingHandler.cs

```diff
@@ -33,7 +33,7 @@ internal class RetryDelegatingHandler : DefaultDelegatingHandler
         private bool _deviceReceiveMessageEnabled;
         private bool _isDisposing;
         private bool _isAnEdgeModule = true;
-        private long _isOpened; // store the opened status in an int which can be accessed via Interlocked class. opened = 1, closed = 0.
+        private long _isOpen; // store the open/closed status in an int which can be accessed via Interlocked class. open = 1, closed = 0.
```

Comment:
Is there any chance this could be made to be a boolean? I'm not that familiar with the ```Interlocked``` class that uses this

Reply:
Interlocked is similar to AtomicInteger in Java, wherein it allows for atomic operations to be executed on the variable location supplied. This works better in C# than using the "volatile" keyword to access the latest value of the variable from memory.
Interlocked, unfortunately, doesn't work with bools so this needs to be defined as a long.